addons/pos_hr/static/src/js/CashierName.js

```diff
@@ -14,8 +14,8 @@ odoo.define('pos_hr.CashierName', function (require) {
             }
             //@Override
             get avatar() {
-                if (this.env.pos.config.module_pos_hr) {
-                    const cashier = this.env.pos.get_cashier();
+                const cashier = this.env.pos.get_cashier();
+                if (this.env.pos.config.module_pos_hr && cashier.id) {
```

Comment:
```suggestion          
                if (this.env.pos.config.module_pos_hr) {
                    const cashier = this.env.pos.get_cashier();
                    if (!(cashier && cashier.id)) {
                        return '';
                    }
```

Let's not fallback on the super call because that one assumes the id to be from `res.users`.

Reply:
Done